.github/workflows/continuous-integration.yml

```
@@ -15,7 +15,7 @@ jobs:
       - name: Execute linters and test suites
         run: ./docker/cibuild
       - name: Upload All coverage to Codecov
-        uses: codecov/codecov-action@v4.1.1
+        uses: codecov/codecov-action@v5.4.2
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./coverage.xml
```

Per v5 migration guide: 

```suggestion
          files: ./coverage.xml
```